tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/portal/PortalRuntimeComposite.java

```diff
@@ -184,6 +283,33 @@ else if( e.getSource().equals( nameField ) )
         updateFields();
 
         validate();
+
+        enableJREControls( true );
+
+        if( getPortalRuntime().getVMInstall() != null )
+        {
+            // check to see if selected VM is in same path as new server
```

Comment:
We shouldn't need this code block now if we remove the "detect bundled jre logic" right?